### ISSUE TITLE
Disable UI buttons for DNS Record in pending state

### ISSUE THREAD

**Issue by humphd** [DevelopingSpace/starchart#361]
When we are creating a DNS Record, it shows immediately in the UI table.  However, it's not appropriate to allow the user to click Delete, Renew, Edit, etc. until the process of creation has completed (e.g., `status=active` or `status=error`).

We should disable these buttons until the `pending` status has been cleared. 

cc @Myrfion

**Comment by Myrfion** [comment]
I guess we should also redirect users from the `domains/:dnsRecordId` route to `domains` if the dnsRecord status is pending, right?

**Comment by humphd** [comment]
Yeah, you shouldn't be able to edit, delete, update, etc. until it's out of `pending`.
